src/ga/individual.rs

```diff
@@ -1,43 +1,41 @@
 use std::fmt::Debug;
-
 use serde::Serialize;
 
-pub trait Gene: Sized + Default + Copy + Debug {}
 
-// Blanket implementaion.
-impl<T: Sized + Default + Copy + Debug> Gene for T {}
+pub trait Chromosome: Sized + Sync + Send + Clone + Default + Debug {}
```

Comment:
`Sized + Sync + Send` -- I followed rules of thumb presented in [this book](https://nostarch.com/rust-rustaceans) - it is reasonable that any type implements these basic traits.

`Clone + Default` -- these traits are required by our current GA implementation.
`Debug` -- required by `StdoutProbe`; potentially this trait bound could be moved to parameter on `Probe` trait.